src/components/Person/Person.jsx

```diff
@@ -1 +1,28 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, partnerName, isMarried, sex } = person;
+  const getPartner = () => {
+    if (isMarried) {
+      if (sex === 'm') {
+        return 'wife';
+      }
+
+      return 'husband';
+    }
+
+    return 'I am not married';
+  };
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {name}</h2>
+
+      {person.age && <p className="Person__age">I am {age}</p>}
+
+      <p className="Person__partner">
+        {partnerName} is my {getPartner()}
```

Comment:
looks strange

<img width="250" alt="Screenshot 2024-09-16 at 20 12 43" src="https://github.com/user-attachments/assets/67116386-b2b9-459d-8570-53281a1c0123">

Reply:
decide before jsx what text will be rendered and just render in jsx. something like that

```
const personPartnerType = sex === 'm' ? 'wife' : 'husband';
const personPartner = isMarried ? `${partnerName} is my ${personPartnerType}` : 'I am not married'
```